.github/workflows/compatibility.yml

```
@@ -11,13 +11,13 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.0-rc.1']
+        woocommerce: [ '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.0' ]
```

I may add `beta` here too, which would have been done before, and leave a comment to explain which range this list should have: 

```suggestion
        # Minimum support version, ..., Latest version, `beta`.
        woocommerce: [ '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.0', '6.1.0', '6.1.1', '6.2.0' ]
```

---

@htdat should we add `beta` as in [here](https://github.com/Automattic/woocommerce-payments/pull/3798/files#diff-5cc2c864012004f60ee3f9078fa1c4a8721c6b841e386c57f12bea58b692e6e2R39)?

---

> should we add `beta` as in [here](https://github.com/Automattic/woocommerce-payments/pull/3798/files#diff-5cc2c864012004f60ee3f9078fa1c4a8721c6b841e386c57f12bea58b692e6e2R39)?

Yes, I see that we have already added it? 

--- 

Also, my suggestion code above missed `beta` at the end, so I've just updated it.

---

Yeah, I was talking about this specific change 👍.